addons/account_edi_ubl_cii/wizard/account_move_send.py

```diff
@@ -237,7 +238,7 @@ def _postprocess_invoice_ubl_xml(self, invoice, invoice_data):
             doc_type_node = f"<cbc:DocumentTypeCode {doc_type_code_attrs}>{doc_type_code_vals['value']}</cbc:DocumentTypeCode>"
         to_inject = f'''
             <cac:AdditionalDocumentReference
-                xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2"
+                xmlns="urn:oasis:names:specification:ubl:schema:xsd:{xmlns_move_type}-2"
```

Comment:
useless diff?

Reply:
Not useless ;) when sending CreditNote, that xmlns thing must be updated to CreditNote too, otherwise the xml won't get accepted by Romania

Reply:
So it's more a bugfix when embedding the file